`panic` on `telepresence uninstall --everything`

**Describe the bug**
`panic` on `telepresence uninstall --everything`

```
$ telepresence uninstall --everything
--everything is deprecated. Please use telepresence helm uninstall
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x60 pc=0x1039fe938]

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).VisitAll(0x140009bf958?, 0x10415051c?)
        github.com/spf13/pflag@v1.0.5/flag.go:275 +0x18
github.com/telepresenceio/telepresence/v2/pkg/client/cli/daemon.(*Request).CommitFlags(0x140000d5810, 0x140002e1200)
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/daemon/request.go:74 +0x44
github.com/telepresenceio/telepresence/v2/pkg/client/cli/cmd.(*HelmCommand).run(0x140003da1b0, 0x140002e1200, {0x140009bfac8?, 0x1?, 0x1?})
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/cmd/helm.go:165 +0x24c
github.com/telepresenceio/telepresence/v2/pkg/client/cli/cmd.(*uninstallCommand).args(0x140009bfb48?, 0x103b3542c?, {0x140003d7f80, 0x0, 0x1})
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/cmd/uninstall.go:54 +0x20c
github.com/telepresenceio/telepresence/v2/pkg/client/cli/cmd.argsCheck.func1(0x140002e1200?, {0x140003d7f80?, 0x1?, 0x1?})
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/cmd/telepresence.go:176 +0x28
github.com/spf13/cobra.(*Command).ValidateArgs(...)
        github.com/spf13/cobra@v1.6.1/command.go:1072
github.com/spf13/cobra.(*Command).execute(0x140002e1200?, {0x140003d7f60?, 0x1?, 0x1?})
        github.com/spf13/cobra@v1.6.1/command.go:885 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x14000354000)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/telepresenceio/telepresence/v2/pkg/client/cli/output.Execute(0x104c68f68?)
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/output/output.go:114 +0x30
github.com/telepresenceio/telepresence/v2/pkg/client/cli.Main({0x104c68f68, 0x140004b2c30})
        github.com/telepresenceio/telepresence/v2@v2.13.0/pkg/client/cli/main.go:68 +0x2a4
main.main()
        github.com/datawire/telepresence-pro/cmd/telepresence/main.go:11 +0x34
```

**Version**
```
Client         : v2.13.0
Root Daemon    : v2.13.0
User Daemon    : v2.13.0
Traffic Manager: v2.13.0
```

--- a/pkg/client/cli/cmd/uninstall.go
+++ b/pkg/client/cli/cmd/uninstall.go
@@ -48,7 +48,7 @@ func (u *uninstallCommand) args(cmd *cobra.Command, args []string) error {
 	if u.everything {
 		ha := &HelmCommand{
 			RequestType: connector.HelmRequest_UNINSTALL,
-			Request:     &daemon.Request{},
+			Request:     daemon.InitRequest(cmd),
 		}
 		fmt.Fprintln(cmd.OutOrStderr(), "--everything is deprecated. Please use telepresence helm uninstall")
 		return ha.run(cmd, args)
@@ -81,6 +81,8 @@ func (u *uninstallCommand) run(cmd *cobra.Command, args []string) error {
 	case u.agent:
 		ur.UninstallType = connector.UninstallRequest_NAMED_AGENTS
 		ur.Agents = args
+	case u.everything:
+		return nil
 	default:
 		ur.UninstallType = connector.UninstallRequest_ALL_AGENTS
 	}

